BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -1049,3 +1049,4 @@
 "NoAccountFoundPleaseLogInAgainIfYouContinueToSeeThisError" = "No account found. Please log in again if you continue to see this error.";
 "ImportError" = "Import error";
 "NoLoginsWereImported" = "No logins were imported";
+"ThisIsNotARecognizedBitwardenServer" = "This is not a recognized Bitwarden server";
```

Comment:
🤔 Should this end with a period? It also doesn't seem to be the same as we have on the Android side.

Reply:
ah good catch! thanks!